bundle/manifests/ibm-auditlogging-operator.clusterserviceversion.yaml

```diff
@@ -40,6 +40,7 @@ metadata:
             "fluentd": {
               "imageRegistry": "quay.io/opencloudio/",
               "pullPolicy": "IfNotPresent",
+              "zenEnabled": false
```

Comment:
Related to other comments: Is a ZenEnabled variable required ... or is it just a temporary until we get the certificates/secrets worked out?